Updates to communicate new raster output units (InVEST #1270) (#158)
natcap/invest.users-guide: source/en/carbon_edge.rst

Final Results
~~~~~~~~~~~~~

> -  **Parameter log**. Each time the model is run, a text (.txt) file will appear in the *Workspace*. The file will list the parameter values for that run and will be named according to the service, the date and time. When contacting NatCap about errors in a model run, please include the parameter log.

Reviewer: ```suggestion
-  **Parameter log**. Each time the model is run, a text (.txt) file will appear in the *Workspace*. The file will list the parameter values for that run and will be named according to the service, date, and time. Please include the parameter log when contacting NatCap about errors in a model run.

```

Reviewer: Added Oxford comma and switched from passive to active voice, per the [UG style guide](https://docs.google.com/document/d/1BHwHDu_I-x0s_2GsbUb4rfVmXMkl7kl97sx2suBTLh8/edit?usp=sharing).

> -  **carbon_map_[Suffix].tif**. A map of carbon stock, with the amount in forest derived from the regression based on distance to forest edge, and the amount in non-forest classes according to the biophysical table. Units are metric tons per hectare (Mg/ha).

Author: Removed "per pixel" and removed the note about how values change with pixel size (no longer relevant since values are no longer per pixel).

> -  **carbon_map_[Suffix].tif**. A map of carbon stock, with the amount in forest derived from the regression based on distance to forest edge, and the amount in non-forest classes according to the biophysical table. Units are metric tons per hectare (Mg/ha).

Reviewer: ```suggestion
-  **carbon_map_[Suffix].tif**. A map of carbon stock, with the amount in forest derived from the regression based on distance to forest edge, and the amount in non-forest classes according to the biophysical table. Units are metric tons per hectare (t/ha).

```

`Mg` notation should be removed all together as "megagrams" are being replaced with "tons". Listing "Mg" as the abbreviation for "metric tons" would likely only add to users' confusion.

Reviewer 2: I agree with Jesse that we should be consistent with "metric tons" and remove Mg, as it's confusing.

> -  **carbon_map_[Suffix].tif**. A map of carbon stock, with the amount in forest derived from the regression based on distance to forest edge, and the amount in non-forest classes according to the biophysical table. Units are metric tons per hectare (Mg/ha).

Reviewer: ```suggestion
-  **carbon_map_[Suffix].tif**. A map of carbon stock, with the amount in forest derived from the regression based on distance to forest edge, and the amount in non-forest classes according to the biophysical table. Units are metric tons per hectare (t/ha).

```
`Mg` notation should be removed all together and replaced with `t` as "megagrams" are being replaced with "tons". Listing "Mg" as the abbreviation for "metric tons" would likely only add to users' confusion.

> -  **aggregated_carbon_stocks_[Suffix].shp**: If service areas of interest are provided by the user, this output shapefile summarizes the total carbon (column *c_sum*) and mean carbon per ha (column *c_ha_mean*) in the area polygons defined. Values are in metric tons (Mg).

Reviewer: ```suggestion
-  **aggregated_carbon_stocks_[Suffix].shp**: If service areas of interest are provided by the user, this output shapefile summarizes the total carbon (column *c_sum*) and mean carbon per hectare (column *c_ha_mean*) in the area defined by polygons. Values are in metric tons (t).

```
For true clarity, "metric tons" ought to be abbreviated as `t`, not as `Mg`. Also, spelled out "hectare" here and edited sentence.

Conservation Biology 15, 1529–1535.

Laurance, W., Lovejoy, T., Vasconcelos, H., Bruna, E., Didham, R.,
Stouffer, P., Gascon, C., Bierregaard, R., Laurance, S., Sampaio, E., 2002.
Ecosystem decay of Amazonian forest fragments: a 22-year

> investigation. Conservation Biology 16, 605–618.

Author: In a previous commit, `2002. ` was somehow converted to `1. ` in my code editor. My theory is, the line beginning with `2002. ` was interpreted as a list item, so I moved `2002.` to the end of the previous line to prevent this from happening in the future.
